ui_src/src/domain/overview/index.js

```diff
@@ -131,7 +132,7 @@ function OverView() {
         getOverviewData();
         setBotImage(localStorage.getItem(LOCAL_STORAGE_AVATAR_ID) || state?.userData?.avatar_id);
         SetUsername(
-            localStorage.getItem(LOCAL_STORAGE_FULL_NAME) !== 'undefined' && localStorage.getItem(LOCAL_STORAGE_FULL_NAME) !== ''
+            localStorage.getItem(LOCAL_STORAGE_FULL_NAME) !== 'undefined' && localStorage.getItem(LOCAL_STORAGE_FULL_NAME) !== ' ' && localStorage.getItem(LOCAL_STORAGE_FULL_NAME) !== ''
```

Comment:
Why did we add this validation? Could it be a situation of a username with a single space?